gateway-ha/src/main/java/com/lyft/data/gateway/ha/clustermonitor/ActiveClusterMonitor.java

```diff
@@ -20,27 +18,28 @@
 import java.util.concurrent.Future;
 import java.util.concurrent.TimeUnit;
 import javax.ws.rs.HttpMethod;
-import lombok.Data;
-import lombok.ToString;
 import lombok.extern.slf4j.Slf4j;
 import org.apache.http.HttpStatus;
 
 @Slf4j
 public class ActiveClusterMonitor implements Managed {
   private static final ObjectMapper OBJECT_MAPPER = new ObjectMapper();
   private static final int BACKEND_CONNECT_TIMEOUT_SECONDS = 15;
-  private static final int MAX_THRESHOLD_QUEUED_QUERY_COUNT = 100;
-
   private static final int MONITOR_TASK_DELAY_MIN = 1;
 
-  @Inject private Notifier emailNotifier;
-  @Inject private GatewayBackendManager gatewayBackendManager;
+  @Inject
+  private List<PrestoClusterStatsObserver> clusterStatsObservers;
```

Comment:
Introducing listeners , which can reactive to the stats collected from the Active Cluster monitor